src/SFA.DAS.EAS.Web/Controllers/EmployerCommitmentsController.cs

```diff
@@ -61,6 +61,10 @@ public async Task<ActionResult> Index(string hashedAccountId)
         [Route("cohorts")]
         public async Task<ActionResult> YourCohorts(string hashedAccountId)
         {
+            if (_featureToggleService.Get<EnhancedApprovals>().FeatureEnabled)
+                return Redirect(_linkGenerator.CommitmentsV2Link($"{hashedAccountId}/unapproved"));
+
+            // otherwise call existing code
```

Comment:
I think we should redirect after the authorisation check.

Reply:
Not needed as its done in V2